redwood-protocol-widget/src/commonMain/kotlin/app/cash/redwood/protocol/widget/RedwoodView.kt

```diff
@@ -13,42 +13,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package app.cash.redwood.treehouse
+package app.cash.redwood.protocol.widget
 
-import app.cash.redwood.protocol.widget.ProtocolMismatchHandler
-import app.cash.redwood.protocol.widget.ProtocolNode
 import app.cash.redwood.ui.UiConfiguration
 import app.cash.redwood.widget.Widget
 import kotlin.native.ObjCName
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.serialization.json.Json
 
-@ObjCName("TreehouseView", exact = true)
-public interface TreehouseView {
+@ObjCName("RedwoodView", exact = true)
+public interface RedwoodView {
   public val children: Widget.Children<*>
   public val uiConfiguration: StateFlow<UiConfiguration>
   public val widgetSystem: WidgetSystem
   public val readyForContent: Boolean
   public var readyForContentChangeListener: ReadyForContentChangeListener?
```

Comment:
The need for these three things is very suspect to me. We shouldn't need any of these in pure Redwood.

Reply:
`RedwoodView` has been introduced in https://github.com/cashapp/redwood/pull/1435 and no longer includes these three things, nor `reset()`.